sycl/plugins/xrt/pi_xrt.cpp

```diff
@@ -1325,9 +1364,23 @@ pi_result xrt_piQueueCreate(pi_context context, pi_device device,
                             pi_queue_properties properties, pi_queue *queue) {
   assert_valid_obj(context);
   assert_valid_obj(device);
-  // TODO(XRT): : properties not handled
-
-  *queue = make_ref_counted<_pi_queue>(context, device).give_externally();
+  if (properties) {
+    if (properties & PI_QUEUE_FLAG_PROFILING_ENABLE)
+      std::cerr
+          << "warning: support for profiling is only partial. pi_xrt is fully "
+             "synchonous so it will detect when an event is completed only "
+             "when the user waits on the event. if the user waits on the event "
```

Comment:
```suggestion
             "synchronous so it will detect when an event is completed only "
             "when the user waits for the event. If the user waits on the event "
```